syntax/basic/literal.vim

```diff
@@ -23,6 +23,8 @@ syntax region  typescriptTemplate
   \ nextgroup=@typescriptSymbols
   \ skipwhite skipempty
 
+syntax match typescriptTaggedTemplate /\<\K\k*\ze`/ nextgroup=typescriptTemplate
```

Comment:
Please don't do this. It will overlap with typescriptIdentifier and make backtracking more often.

It also doesn't handle case like funcCall()``

Reply:
Ah, I think I'm at the limit of my vim syntax knowledge then so I'm not sure where to put it :-).

So far I'm using it and the funcCall()`` case gets handled tho.  Must be handled by the usual typescriptTemplate match.